content/post/why-i-joined-sourced-francesc.md

```diff
@@ -0,0 +1,117 @@
+---
+author: campoy
+date: 2018-01-15
+title: "Why did I join source{d}? - Francesc Campoy"
+draft: true
+description: "The first post of a series on why multiple employees joined source{d}. This one is by Francesc Campoy."
```

Comment:
The article lacks image, what breaks our listings. What if we ask our designer for a header to this kind of posts?

Reply:
I copy pasted and totally forgot about the images, good catch!
Adding them now.